3_visualize.yml

```diff
@@ -29,6 +32,17 @@ targets:
   src/assets/anomaly_peaks.svg:
     command: file.copy(
       to = target_name,
-      out = '3_visualize/out/anomaly_peaks.svg', 
+      from = '3_visualize/out/anomaly_peaks.svg', 
       overwrite = TRUE)
       
+  public/date_peaks.csv:
+    command: gwl_to_peak(
+      target_name, 
+      gw_anomaly_data_w_colors)
+    
+  public/gw_sites.csv:
```

Comment:
I like how you have these being saved directly to `public/` instead of in a pipeline folder and then copying over ... seems like we should switch the `anomaly_peaks.svg` target to do something similar at some point in the future.

Reply:
^ this is not a change request just to clarify :) just me thinking out loud